src/App.tsx

```diff
@@ -16,9 +18,15 @@ interface IStates {
 interface IProps {
   userHasAuthenticated: (boolean) => void,
   history: History,
+  renderChildren?: boolean
 }
 
-class App extends Component<IProps, IStates> {
+export class App extends Component<IProps, IStates> {
+  static defaultProps: IProps = {
+    userHasAuthenticated: () => ({}),
```

Comment:
The return type of `userHasAuthenticated` is a `void`, not an empty object. Is it correct?